python/GafferUITest/GLWidgetTest.py

```diff
@@ -75,7 +75,7 @@ def testOverlayWidgetAt( self ) :
 
 		w.setVisible( True )
 
-		self.waitForIdle( 10000 )
+		self.waitForIdle( 1000000 )
```

Comment:
This test now takes a full 8 seconds on my Linux machine. Do we have any idea why we need to wait that long?

Reply:
Dropped in the latest push.